src/transformation/visitors/break-continue.ts

```diff
@@ -8,7 +8,7 @@ import { findScope, ScopeType } from "../utils/scope";
 export const transformBreakStatement: FunctionVisitor<ts.BreakStatement> = (breakStatement, context) => {
     const breakableScope = findScope(context, ScopeType.Loop | ScopeType.Switch);
     if (breakableScope?.type === ScopeType.Switch) {
-        return lua.createGotoStatement(`____switch${breakableScope.id}_end`);
+        return undefined;
     } else {
```

Comment:
Maybe add a comment stating switch breaks are handled by the switch tranfromation.